docs/guides/admin/email/gmail.md

```diff
@@ -1,9 +1,11 @@
 ![SeAT](http://i.imgur.com/aPPOxSK.png)
 
-## introduction
+# Email Setup - Using Gmail
+
+## Introduction
 SeAT requires email to be setup to allow for things like password reminders and notifications to be sent. This guide will attempt to describe how to go about setting up your email using [gmail](https://www.gmail.com/) as an SMTP.
```

Comment:
This is not fully correct anymore. The Mail is only needed *if* you send mail notification. Passwords won't be sended out anymore